lib/share_progress/button.rb

```diff
@@ -21,7 +35,8 @@ def destroy(id)
 
       def all(options={})
```

Comment:
ahh yes I never responded. It seemed to me that it would be better not to have a default here, but rather to let the API set a default (which it does, ten and ten). It felt weird to re-set the defaults, but idk if it matters.